### ISSUE TITLE
Client crash connecting to server

### ISSUE THREAD

**Issue by RogueRaiden** [DArkHekRoMaNT/CommonLib#3]
### Mod Version

2.1.0

### Game Version

1.18.3

### OS

Windows, Linux

### Play mode

Multiplayer

### Problem

Client crash attempting to connect to server

### Logs

```bash
Running on 64 bit Windows with 16 GB RAM 
Game Version: v1.18.3 (Stable)
5/14/2023 8:04:33 PM: Critical error occurred in the following mod: commonlib@2.1.0
Loaded Mods: aculinaryartillery@1.0.12, ancienttools@1.5.4, canmarket@0.2.7, chiseltools@1.8.4, instruments@1.2.0, mannequinstand@1.0.1, meteoricexpansion@1.2.4, leadroof@1.3.2, primitivesurvival@3.1.9, tprunes@1.0.8, temporalstormgear@1.0.1, thebasics@4.0.0-pre.1, game@1.18.3, wildcraft@1.6.6, beehivekiln@1.2.1, blocklog@1.0.0, calendar@1.0.0, carryon@1.1.0, commonlib@2.1.0, expandedfoods@1.6.5, fairplayguardian@1.1.1, sailboat@1.1.0, naturalnight@1.0.0, petai@1.7.7, claywheel@1.1.0, prospectorinfo@4.3.0, quietermusic@1.0.0, rifteye@0.1.3, rhadditions@3.0.5, scarecrow@1.2.3, servertags@1.1.0, stonebakeoven@1.0.1, targetDotSmall@1.1.0, creative@1.18.3, vsquest@0.4.1, survival@1.18.3, wildfarmingrevival@1.0.5, xlib@0.7.1, onebedsleeping@2.2.0, bullseye@2.5.0-rc.1, cats@1.7.0, doffanddonagain@3.0.0, em@2.1.1, playercorpse@1.7.1, simplewinddirection@1.0.2, stonequarry@3.0.0, wolftaming@1.7.3, xinvtweaks@1.5.3, xskills@0.7.1, bricklayers@2.0.0
System.ArgumentNullException: Value cannot be null.
Parameter name: key
   at System.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Collections.Generic.Dictionary`2.FindEntry(TKey key)
   at System.Collections.Generic.Dictionary`2.TryGetValue(TKey key, TValue& value)
   at CommonLib.Config.ConfigManager.<>c__DisplayClass10_0.<StartPre>g__OnSyncConfigPacketReceived|4(SyncConfigPacket packet) in CommonLib/Config/ConfigManager.cs:line 146
   at Vintagestory.Client.NoObf.NetworkChannel.<>c__DisplayClass12_0`1.<SetMessageHandler>b__0(Packet_CustomPacket p) in VintagestoryLib\Client\API\NetworkChannel.cs:line 74
   at Vintagestory.Client.NoObf.NetworkAPI.HandleCustomPacket(Packet_Server packet) in VintagestoryLib\Client\API\NetworkAPI.cs:line 108
   at Vintagestory.Client.NoObf.ClientMain.ExecuteMainThreadTasks(Single deltaTime) in VintagestoryLib\Client\ClientMain.cs:line 747
   at Vintagestory.Client.GuiScreenConnectingToServer.RenderToDefaultFramebuffer(Single dt) in VintagestoryLib\Client\MainMenu\Screens\GuiScreenConnectingToServer.cs:line 304
   at Vintagestory.Client.ScreenManager.Render(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 724
   at Vintagestory.Client.ScreenManager.OnNewFrame(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 627
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e) in VintagestoryLib\Client\ClientPlatform\GameWindow.cs:line 126
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 479
   at OpenTK.GameWindow.DispatchRenderFrame() in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 454
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 375
   at Vintagestory.Client.ClientProgram.Start(ClientProgramArgs args, String[] rawArgs)
   at Vintagestory.ClientNative.CrashReporter.Start(ThreadStart start) in VintagestoryLib\Client\ClientPlatform\ClientNative\CrashReporter.cs:line 93
```

**Comment by DArkHekRoMaNT** [comment]
Can you send me client-main.txt, client-debug.txt and server-main.txt?

**Comment by RogueRaiden** [comment]
[server-main.txt](https://github.com/DArkHekRoMaNT/CommonLib/files/11473416/server-main.txt)
[client-main.txt](https://github.com/DArkHekRoMaNT/CommonLib/files/11473418/client-main.txt)
[client-debug.txt](https://github.com/DArkHekRoMaNT/CommonLib/files/11473419/client-debug.txt)

**Comment by DArkHekRoMaNT** [comment]
Fixed in 2.1.1
